dirsrvtests/tests/suites/retrocl/retrocl_indexing_test.py

```diff
@@ -69,6 +83,76 @@ def test_indexing_is_online(topo):
     assert not topo.standalone.searchAccessLog('Partially Unindexed Filter')
 
 
+def test_reindexing_retrochangelog(topology_m1):
+    """ Test that the retro changelog can be successfully reindexed
+
+        :id: 26d8df59-0886-4098-8d2c-b9337accc908
+        :setup: Supplier instance
+        :steps:
+            1. Enable retro changelog and restart instance
+            2. Create test entries
+            3. Create index
+            4. Reindex retro changelog
+            5. Verify indexed search
+            6. Clean up test entries
+        :expectedresults:
+            1. Success
+            2. Success
+            3. Success
+            4. Success
+            5. Success
+            6. Success
+        """
+
+    log.info('Enable retro changelog and restart instance')
+    retrocl = RetroChangelogPlugin(topology_m1.ms["supplier1"])
+    retrocl.enable()
+    topology_m1.ms["supplier1"].restart()
+
+    log.info('Creating test entries')
+    for count in range(MAX_OTHERS):
+        name = "{}{}".format(OTHER_NAME, count)
+        topology_m1.ms["supplier1"].add_s(
+            Entry(("cn={},{}".format(name, DEFAULT_SUFFIX), {
+            'objectclass': "top person".split(),
+            'sn': name,
+            'cn': name})),
+            escapehatch='i am sure')
```

Comment:
We shouldn't use `Entry` struct as it'll be deprecated eventually.

When we need to add users, we can use `UserAccounts` or generate an ldif and import it.